server/comms/comms_test.go

```diff
@@ -610,7 +610,7 @@ func TestOnline(t *testing.T) {
 
 	keyPath := filepath.Join(tempDir, "rpc.key")
 	certPath := filepath.Join(tempDir, "rpc.cert")
-	pongWait = 100 * time.Millisecond
```

Comment:
Only thing with this is that we have a three-ping sleep down here: https://github.com/decred/dcrdex/pull/493/files#diff-a19499504b77f7a1f4e6498dbb82929cR707-R708, so it's in our interest to reduce that to a couple pings (@buck54321 I was never really sure why waiting for more than one ping was needed, can you clarify?) or maybe settle on a 500ms pong wait instead.

Reply:
I was probably being sloppy around the edges. 2 pings at 500 ms seems better.